ports/shaderwriter/portfile.cmake

```
@@ -29,6 +29,7 @@ vcpkg_cmake_configure(
         -DSDW_UNITY_BUILD=ON
 )
 
+vcpkg_copy_pdbs()
 vcpkg_cmake_install()
 vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/shaderwriter)

```

For the first suggestion:
```suggestion
vcpkg_cmake_config_fixup(PACKAGE_NAME ShaderWriter CONFIG_PATH lib/cmake/shaderwriter DO_NOT_DELETE_PARENT_CONFIG_PATH)
vcpkg_cmake_config_fixup(PACKAGE_NAME SdwCompilers CONFIG_PATH lib/cmake/shaderwriter)
```

---

Well, I've actually reworked the install script in the project, hence this change won't be needed.